beacon-chain/state/interfaces.go

```diff
@@ -283,3 +300,19 @@ type WriteOnlySyncCommittee interface {
 	SetCurrentSyncCommittee(val *ethpb.SyncCommittee) error
 	SetNextSyncCommittee(val *ethpb.SyncCommittee) error
 }
+
+type WriteOnlyPendingBalanceDeposits interface {
+	AppendPendingBalanceDeposit(index primitives.ValidatorIndex, amount uint64) error
```

Comment:
dumb question here why is this in its own interface instead of writeOnlyElectra?

Reply:
I think I had split something out for maxEB, but I don't remember. There is no reason, I guess